pkg/util/port/portlist/netstat_windows.go

```diff
@@ -0,0 +1,283 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build windows
```

Comment:
You don't **need** the build tag, it is added implicitly since the file has `_windows` suffix